packages/neos-ui/src/Containers/LeftSideBar/NodeTree/Node/index.js

```diff
@@ -49,6 +63,26 @@ export default class Node extends PureComponent {
         this.handleNodeLabelClick = this.handleNodeLabelClick.bind(this);
     }
 
+    componentDidUpdate() {
+        this.scrollFocusedNodeIntoView();
+    }
+
+    scrollFocusedNodeIntoView() {
+        if (this.isFocused()) {
+            const scrollingElement = findScrollingParent(this.domNode);
+            if (scrollingElement) {
+                const nodePosition = this.domNode.getBoundingClientRect().top;
```

Comment:
This should get a clearer name as `scrollingElementPosition` is the BoundingClientRect it should be make clear that nodePosition has something to do with top.